### ISSUE TITLE
BOINC 7 causes Windows 7 64bit explorer crashes

### ISSUE THREAD

**Issue by romw** [BOINC/boinc#1169]
**Reported by Ignitor on 10 Jun 42522495 01:49 UTC**
Starting from version 7 (I can confirm for version 7.0.25 and 7.0.28), BOINC seems to cause explorer.exe crashes on Windows 7 64 bit. I could not determine how to reproduce the crashes yet because they seem to be quite random. However, I got the feeling that the crashes occur especially at the moment I get back to the computer and BOINC pauses.

This is the content of my stderrdae.txt:
```
Another instance of BOINC is running.
GLE: Another instance of BOINC is running.
GLE: Another instance of BOINC is runni
Another instance of BOINC is running.
GLE: Another instance of BOINC is running.
GLE: Another instance of BOINC is runni
Another instance of BOINC is running.
GLE: Another instance of BOINC is running.
GLE: Another instance of BOINC is runni
Another instance of BOINC is running.
GLE: Another instance of BOINC is running.
GLE: Another instance of BOINC is runni
```
As you can see, sometimes, it doesn't even finish to write the whole string ("ng." is missing). The other stderr files are empty. The stdoutgui.txt contains some error messages:
```
[22:31:56](11/04/11) TRACE [init_poll(): connected

22:31:56: Error: Memory VFS already contains file 'webexternallink.xpm'!
22:31:56: Error: Memory VFS already contains file 'nvidiaicon.xpm'!
22:31:56: Error: Memory VFS already contains file 'atiicon.xpm'!
22:31:56: Error: Memory VFS already contains file 'multicore.xpm'!
22:33:44: Error: Memory VFS already contains file 'webexternallink.xpm'!
22:33:44: Error: Memory VFS already contains file 'nvidiaicon.xpm'!
22:33:44: Error: Memory VFS already contains file 'atiicon.xpm'!
22:33:44: Error: Memory VFS already contains file 'multicore.xpm'!
22:33:44: Error: Memory VFS already contains file 'webexternallink.xpm'!
22:33:44: Error: Memory VFS already contains file 'nvidiaicon.xpm'!
22:33:44: Error: Memory VFS already contains file 'atiicon.xpm'!
22:33:44: Error: Memory VFS already contains file 'multicore.xpm'!
22:33:56: Error: Can not enumerate files in directory 'C:\Program Files (x86)\BOINC\\skins\' (error 3: das System kann den angegebenen Pfad nicht finden.)
22:33:56: Error: Can not enumerate files in directory 'C:\Program Files (x86)\BOINC\\skins\' (error 3: das System kann den angegebenen Pfad nicht finden.)
[04/19/12 11:23:25](3228]:) TRACE [init_asynch() boinc_socket: 712

[04/19/12 11:23:25](5852]:) TRACE [5852]: init_asynch() connect: -1
```
I'll attach the stdoutdae.txt.

Let me know if you need further information.

There is also a forum thread regarding this problem: http://boinc.berkeley.edu/dev/forum_thread.php?id=7566

Migrated-From: http://boinc.berkeley.edu/trac/ticket/1197

**Comment by romw** [comment]
**Commented by Ignitor on 25 Dec 42522524 16:19 UTC**
Well, I couldn't attach the whole stdoutdae.txt because I got an error message like
"```Akismet says it is spam. Maximum number of external links exceeded```". So I attached only the part of the last crash and modified the URLs within the file.

**Comment by romw** [comment]
**Commented by Ignitor on 24 Oct 43086992 09:43 UTC**
I get the feeling that the problem is related to the graphics card. I disabled the GPU (using `<no_gpus>` in the client configuration as described at http://boinc.berkeley.edu/wiki/Client_configuration) and since then, ~~the explorer crashes have stopped~~ the number of explorer crashes decreased.

My guess is that Aero is using the GPU simultaneously with BOINC and this doesn't work well.

I got an nVidia !GeForce GTX 560 Ti (driver version 9.18.13.697).
